packages/thirdweb/src/utils/encoding/helpers/is-hex.ts

```diff
@@ -22,13 +20,27 @@ export function isHex(
   value: unknown,
   options: IsHexOptions = {},
 ): value is Hex {
-  if (!value) {
-    return false;
-  }
-  if (typeof value !== "string") {
-    return false;
+  const evenLengthHex = (() => {
+    if (
+      typeof value === "string" &&
+      value.slice(0, 2) === "0x" &&
+      value.length % 2 !== 0
+    ) {
+      return ox__Hex.padLeft(value as `0x${string}`, 2);
+    }
+    return value;
+  })();
```

Comment:
I think assertion should still work without this?

Reply:
I think we can just use `Hex.validate()` here (or am I imagining it being a thing?)

Reply:
oh you literally commented the same thing on the file lol